spec/models/feedbook/answer_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
```

Comment:
If you add a `.rspec` file with `--require rails_helper` in it, you won't need to have this `require` in every spec file.